person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_reader :id, :parent_permission
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = !!parent_permission
+  end
+
+  def of_age?
+    is_of_age = false
+    is_of_age = true if @age >= 18
+    is_of_age
+  end
```

Comment:
- [ ] [OPTIONAL] In this method it is not needed to first set a variable to `false` and then check with an if statement and return the variable, it can be written like below.
```
  def of_age?
    @age >= 18
  end
```